topics/statistics/tutorials/machinelearning/slides.html

```diff
@@ -0,0 +1,224 @@
+    - Each sample/record in the training data has a category/class/label.
+    - A machine learning algorithm learns features from the training data and do predictions on the test data.
+contributors:
+  - anuprulez
+---
+
+# Contents
+
+- What is machine learning?
+- Types of machine learning
+- Techniques for
+    - Hyperparameter optimisation
+    - Learning and evaluation of models
+- Various applications of machine learning
+
+---
+
+# Machine learning
+
+.pull-left[
+- Learns patterns from data
+- Comprises of different fields
+    - Linear algebra, statistics and probability
+    - Programming
+    - Data analysis
+    - Visualization
+- Applicable to data from multiple fields - protein and DNA sequences, weather data, stock and house prices, images ...
+
+]
+
+.pull-right[ ![Machine learning]({{site.baseurl}}/topics/statistics/tutorials/machinelearning/images/it1.png)]
+
+---
+
+# Variants of ML
+
+.center[ ![ML variants]({{site.baseurl}}/topics/statistics/tutorials/machinelearning/images/it2.png)]
+
+---
+
+# Classification
+
+.pull-left[
+- Supervised learning
+- Learn/predict classes or targets 
+- Find decision boundary
+- Linear and non-linear boundaries
+- Algorithms are classifiers
+- Examples
+    - Tumor or no tumor
+    - Rain or no rain
+    - ...
+]
+
+.pull-right[ ![Classification]({{site.baseurl}}/topics/statistics/tutorials/machinelearning/images/it3.png)]
+
+---
+
+# Classification dataset
+
+- Breast tumor dataset - Features and target
+
+.center[  ![Classification dataset]({{site.baseurl}}/topics/statistics/tutorials/machinelearning/images/it4.png)]
+
+---
+
+# Regression
+
+.pull-left[
+- Supervised learning
+- Targets are real numbers
+- Find fitting curve 
+- Linear or non-linear curves
+- Algorithms are regressors
+- Examples:
+    - Temperature forecast
+    - Stock/house prices
+    - ...
+
+]
+
+.pull-right[ ![Regression]({{site.baseurl}}/topics/statistics/tutorials/machinelearning/images/it5.png)]
+
+---
+
+# Regression dataset
+
+- Body fat dataset - features and target
+
+.center[ ![Ensemble model]({{site.baseurl}}/topics/statistics/tutorials/machinelearning/images/it6.png)]
+
+---
+
+# Hyperparameter optimisation
+
+.pull-left[
+- Grid search
+- Random search
+
+]
+
+.pull-right[ ![Hyperparameter optimisation]({{site.baseurl}}/topics/statistics/tutorials/machinelearning/images/it7.png)]
+
+---
+
+# Learning and evaluation
+
+.pull-left[
+- K-fold cross-validation
+- Dataset in K equal parts
+- Part == fold
+- Learn on training set
+- Evaluate on validation set
+
+]
+
+.pull-right[ ![Learning and evaluation]({{site.baseurl}}/topics/statistics/tutorials/machinelearning/images/it8.png)]
+
+---
+
+# Learning and evaluation
+
+.pull-left[
+- Training and test sets
+- Learn on training set
+- Evaluate on test set
+]
+
+.pull-right[ ![Learning and evaluation]({{site.baseurl}}/topics/statistics/tutorials/machinelearning/images/it9.png)]
+
+---
+
+# Applications of machine learning
+
+.pull-left[
+- BioInformatics
```

Comment:
```suggestion
- Bioinformatics
```

Reply:
Or is there a reason for the capital I?
You have other 'Informatics' arias listed below. And here are only Bioinf examples.